cos-fleetshard-operator/src/main/java/org/bf2/cos/fleetshard/operator/connector/ConnectorController.java

```diff
@@ -581,6 +582,11 @@ private UpdateControl<ManagedConnector> handleAugmentation(ManagedConnector conn
                     .withBlockOwnerDeletion(true)
                     .build()));
 
+            if (LOGGER.isDebugEnabled()) {
```

Comment:
This must be removed as it may lead to secrets being leaked out in the logs

Reply:
Oh OK will do